### PR TITLE
Task/03 components Serhii Shevchenko

### PR DESCRIPTION
**Pull request by greyzerua**
- [x] Mentor is added to pr
- [x] AC copied to pr description
- [x] Link to deployed version is added to pr https://greyzerua.github.io/fe-react-2024/
- [x] The ProductsList component correctly receives product data as props.
- [x] ProductCard component created and renders a single product.
- [x] Header and Footer elements are present.
- [x] ProductList displays right on mobile devices
- [x] The Header component has clickable elements that toggle the display between the About and ProductsList components.
- [x] State management is handled in App.tsx, with state determining which component to show.
- [x] LocalStorage added to save items in card after page realod

photo_2024

### CODE REVIEW

File: src/components/app-link/app-link.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    dataHover?: string;
+    currentPage: EAppPage;
+    page: EAppPage;
+    onPageChange: (page: EAppPage) => void;
+};
+
+export const AppLink = ({ className = '', onPageChange, page, currentPage, children, dataHover, ...restProps }: Props) => {
+    const href = APP_LINK_URLS[page];
+
+    const isActive = page === currentPage;
```
Comment (vitalii-mel-work) [inline src/components/app-link/app-link.tsx]: if use only one , not need const here

File: src/components/products-card/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { ProductsCard } from './products-card';
```
Comment (vitalii-mel-work) [inline src/components/products-card/index.ts]: rm it

Comment (vitalii-mel-work) [reply src/components/products-card/index.ts]: Screenshot 2024-05-03 at 12 17 20

**Review by vitalii-mel-work — APPROVED** [review]

File: src/components/app-link/app-link.tsx (hunk 1 above)
Comment (greyzerua) [reply src/components/app-link/app-link.tsx]: Fixed

File: src/components/products-card/index.ts (hunk 2 above)
Comment (greyzerua) [reply src/components/products-card/index.ts]: fixed

**Review by vitalii-mel-work — APPROVED** [review]

**Pull request merged**